src/components/WalletFeatures/WithdrawNFTFunction.tsx

```diff
@@ -0,0 +1,124 @@
+import { Link, ImmutableXClient, ImmutableMethodResults, ERC721TokenType, ImmutableRollupStatus  } from '@imtbl/imx-sdk';
+import { useEffect, useState } from 'react';
+import { Button } from 'react-bootstrap';
+
+interface WithdrawNFTProps {
+  client: ImmutableXClient,
+  link: Link,
+  wallet: string
+}
+
+const WithdrawNFTFunction = ({client, link, wallet}: WithdrawNFTProps) => {
+  // withdrawals
+  const [preparingWithdrawals, setPreparingWithdrawals] = useState<ImmutableMethodResults.ImmutableGetWithdrawalsResult>(Object);
+  const [readyWithdrawals, setReadyWithdrawals] = useState<ImmutableMethodResults.ImmutableGetWithdrawalsResult>(Object);
+  const [completedWithdrawals, setCompletedWithdrawals] = useState<ImmutableMethodResults.ImmutableGetWithdrawalsResult>(Object);
+  // nft
+  const [depositTokenId, setDepositTokenId] = useState('');
+  const [depositTokenAddress, setDepositTokenAddress] = useState('');
+  const [prepareTokenId, setPrepareTokenId] = useState('');
+  const [prepareTokenAddress, setPrepareTokenAddress] = useState('');
+  const [completeTokenId, setCompleteTokenId] = useState('');
+  const [completeTokenAddress, setCompleteTokenAddress] = useState('');
```

Comment:
Too many states that doesn't make sense. TokenId and TokenAddress is always be as a pair in this case.